aptos-move/e2e-tests/src/account.rs

```diff
@@ -26,7 +26,7 @@ use move_deps::move_core_types::{
 use vm_genesis::GENESIS_KEYPAIR;
 
 // TTL is 86400s. Initial time was set to 0.
-pub const DEFAULT_EXPIRATION_TIME: u64 = 40_000;
```

Comment:
This expiration time is way too low. Wouldn't last more than a couple of epochs in tests if each epoch is 7200 seconds.